pkg/apis/serving/v1alpha1/service_types.go

```diff
@@ -234,6 +236,21 @@ func (ss *ServiceStatus) InitializeConditions() {
 	serviceCondSet.Manage(ss).InitializeConditions()
 }
 
+// MarkConfigurationNotOwned surfaces a failure via the ConfigurationsReady
+// status noting that the Configuration with the name we want has already
+// been created and we do not own it.
+func (ss *ServiceStatus) MarkConfigurationNotOwned(name string) {
+	serviceCondSet.Manage(ss).MarkFalse(ServiceConditionConfigurationsReady, "NotOwned",
```

Comment:
This "NotOwned" is same constant throughout code and tests. Can we put it as a constant in a shared place and reuse?

Reply:
We've (subconsciously) avoided this style for Reason (vs. Type) because we don't want folks to treat it like an enum with a bounded space.  I think to achieve what you suggest would require us to export constants alongside our types, which further advertise this in ways that suggest folks can depend on the constant, which makes me uneasy.

So I think I'll consciously not do this for now, but I'm happy to discuss this holistically next week if you have opinions?

Reply:
It does not have to be a bounded "enum". But if a reason is has a multitude of applications, strictly from the code perspective it's be nice to avoid repetitions.
Same, applies, for the Sprintf() formatting string.
But we we consciously want to keep this unconscious way, I'm fine :)